libs/s25main/mapGenerator/HeadQuarters.h

```diff
@@ -91,19 +98,17 @@ namespace rttr { namespace mapGenerator {
      * @param index player index for the HQ
      * @param area area to place the HQ in
      *
-     * @returns true if a HQ was placed fro the specified player, false otherwise.
+     * @throw runtime_error
      */
     template<class T_Container>
-    bool PlaceHeadQuarter(Map& map, int index, const T_Container& area)
+    void PlaceHeadquarter(Map& map, int index, const T_Container& area)
     {
-        auto positions = FindHqPositions(map, area);
+        const auto& positions = FindHqPositions(map, area);
```

Comment:
Huh? Why the const-ref on a return value by-value?

Reply:
> One new issue and the "harber" typo ;)
> 
> However I'd leave this open for bit the discussion in save intervals, but showing minutes and seconds there should be added anyway for UX.

Displaying minutes & seconds how long it took to create the map until it failed?

Reply:
Don't mind me, didn't have enough coffee. Wrong PR :D

Reply:
> Don't mind me, didn't have enough coffee. Wrong PR :D

Dangerous! Cheers!